src/victorops.coffee

```diff
@@ -187,6 +187,8 @@ class VictorOps extends Adapter
       for item in data.PAYLOAD.TIMELINE_LIST
         try
           if item.ALERT
+            @robot.brain.set item.ALERT["INCIDENT_NAME"], item.ALERT
```

Comment:
Hey @charlie-vo, yes, the key-value store would keep growing. It might make sense for me to add some sort of cleanup to the brain down the road so that the brain doesn't grow super huge. 

I believe it would only survive hubot restarts if something like [hubot-redis-brain](https://github.com/hubot-scripts/hubot-redis-brain) was enabled. 

The goal here is to make Incidents available to scripts. If there is a better way to do that, I am open to it.